content/03.methods.md

```diff
@@ -2,13 +2,32 @@
 
 ### Datasets
 
-#### bioRxiv
-1. Describe how bioRxiv was obtained
-2. Describe metadata statistics on bioRxiv (number of preprints, number of preprints with multiple versions)
+#### BioRxiv
+BioRxiv [@doi:10.1101/833400] is a repository that contains biological and biomedical research preprints.
+We downloaded an xml snapshot of this repository on February 3, 2020 and this snapshot contained 98,023 total preprints.
+26,905 out of 98,023 are preprints that contain more than one version as bioRxiv allows users to upload revisions and corrections.
+We fixed this duplication issue by focusing only on the latest version of preprint. 
+Preprints in this snapshot can fall into one of the twenty-nine different categories with neuroscience being the most popular (Figure {@fig:biorxiv_categories}).

```

Comment:
```suggestion
Preprints in this snapshot were grouped into one of twenty-nine different categories.
```
Text and figure should go in results if you want to start getting that together: "with neuroscience being the most popular (Figure {@fig:biorxiv_categories})."